symfony/plugins/orangehrmAdminPlugin/test/Service/EmploymentStatusServiceTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Admin\Tests\Service;
+
+use OrangeHRM\Admin\Dao\EmploymentStatusDao;
+use OrangeHRM\Admin\Dto\EmploymentStatusSearchFilterParams;
+use OrangeHRM\Admin\Service\EmploymentStatusService;
+use OrangeHRM\Config\Config;
+use OrangeHRM\Entity\EmploymentStatus;
+use OrangeHRM\Tests\Util\TestCase;
+use OrangeHRM\Tests\Util\TestDataService;
+use Exception;
+
+/**
+ * @group Admin
+ * @group Service
+ */
+class EmploymentStatusServiceTest extends TestCase
+{
+
+    private $employmentStatusService;
+    protected $fixture;
+
+    /**
+     * Set up method
+     * @throws Exception
+     */
+    protected function setUp(): void
+    {
+        $this->employmentStatusService = new EmploymentStatusService();
+        $this->fixture = Config::get(Config::PLUGINS_DIR) . '/orangehrmAdminPlugin/test/fixtures/EmploymentStatusDao.yml';
+        TestDataService::populate($this->fixture);
+    }
+
+    public function testDeleteEmploymentStatus(): void
+    {
+        $toBeDeletedEmploymentStatusIds = array(1, 2);
```

Comment:
[minor] convert `array()` syntax to `[]`
use IDE https://intellij-support.jetbrains.com/hc/en-us/community/posts/360010152639-How-to-convert-traditional-list-syntax-to-short-syntax-across-whole-project-

Reply:
I chenged it in all test files. Dao and Service (5x)